dbms/src/Storages/Page/stress/workload/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 PingCAP, Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include_directories (${CMAKE_CURRENT_BINARY_DIR})
+
+set (page-workload-src HeavyMemoryCostInGC.cpp HeavyRead.cpp HeavySkewWriteRead.cpp HeavyWrite.cpp HighValidBigFileGC.cpp HoldSnapshotsLongTime.cpp Normal.cpp 
+     PageStorageInMemoryCapacity.cpp ThousandsOfOffset.cpp MainEntry.cpp Normal.cpp PageStorageInMemoryCapacity.cpp PSBackground.cpp PSRunnable.cpp PSStressEnv.cpp PSWorkload.cpp)
+
+add_library (page-workload-lib ${page-workload-src})
+target_link_libraries (page-workload-lib dbms clickhouse_functions clickhouse-server-lib)
+target_compile_options (page-workload-lib PRIVATE -Wno-format -lc++)
+
+add_executable (page-workload Main.cpp ${page-workload-src})
+target_link_libraries (page-workload dbms clickhouse_functions clickhouse-server-lib)
+target_compile_options (page-workload PRIVATE -Wno-format -lc++)
```

Comment:
Seems the binary `page_workload` is useless. Do you think we still need this standalone binary?  @jiaqizho

Reply:
It would increase the compile time of `tiflash` but can decrease the compile when we just want to run `page-workload` only. So maybe we can keep it just like `dt-workload`?

Reply:
In fact, I want to remove the binary of `dt-workload`...

Reply:
ok, I will remove both of them.

Reply:
ok for remove it.

Reply:
done

Reply:
Please delete useless code files as well： https://github.com/pingcap/tiflash/blob/master/dbms/src/Storages/DeltaMerge/tools/workload/Main.cpp

Reply:
done